Source: written by Pierre Lejeune <superheron@gmail.com> for Exherbo
Upstream: No
Reason: Automagic deps are bad
--- configure.ac.orig	2010-10-04 13:03:06.000000000 +0200
+++ configure.ac	2011-03-02 22:35:06.764885001 +0100
@@ -93,18 +93,30 @@
 AC_FUNC_MALLOC
 AC_CHECK_FUNCS([floor memset pow sqrt])
 
+AC_ARG_ENABLE([facebl0r], [  --enable-facebl0r     enable facebl0r plugin (needs opencv)],
+  [facebl0r=${enableval}], [facebl0r=no])
 HAVE_OPENCV=false
-PKG_CHECK_MODULES(OPENCV, opencv >= 1.0.0, [HAVE_OPENCV=true], [true])
-AM_CONDITIONAL([HAVE_OPENCV], [test x$HAVE_OPENCV = xtrue])
-if test x$HAVE_OPENCV = xtrue; then
-#    OPENCV_CFLAGS="$OPENCV_CFLAGS -DOPENCV_PREFIX=`pkg-config opencv --variable=prefix`"
+if test "x${facebl0r}" = xyes; then
+  PKG_CHECK_MODULES(OPENCV, opencv >= 1.0.0, [HAVE_OPENCV=true], [true])
+  if test x$HAVE_OPENCV = xtrue; then
+    OPENCV_CFLAGS="$OPENCV_CFLAGS -DOPENCV_PREFIX=`pkg-config opencv --variable=prefix`"
     AC_DEFINE(HAVE_OPENCV,1,[compiled including opencv])
     AC_DEFINE(OPENCV_DATA_DIR,[${prefix}/share/opencv],opencv data prefix)
+  else
+    AC_MSG_ERROR([*** opencv >= 1.0.0 not found - this program enables optional plugin with the Open Source Computer Vision library >= 1.0.0 http://opencvlibrary.sourceforge.net/ ***])
+  fi
 fi
-AC_SUBST(HAVE_OPENCV)
+AM_CONDITIONAL([HAVE_OPENCV], [test x$HAVE_OPENCV = xtrue])
 
+AC_ARG_ENABLE([scale0tilt], [  --enable-scale0tilt     enable scale0tilt plugin (needs gavl)],
+  [scale0tilt=${enableval}], [scale0tilt=no])
 HAVE_GAVL=false
-PKG_CHECK_MODULES(GAVL, gavl >= 0.2.3, [HAVE_GAVL=true], [true])
+if test "x${scale0tilt}" = xyes; then
+  PKG_CHECK_MODULES(GAVL, gavl >= 0.2.3, [HAVE_GAVL=true], [true])
+  if test x$HAVE_GAVL = xfalse; then
+    AC_MSG_WARN([*** gavl >= 0.2.3 not found - this program enables optional plugin with the gmerlin audio video library >= 0.2.3 http://gmerlin.sourceforge.net/ ***])
+  fi
+fi
 AM_CONDITIONAL([HAVE_GAVL], [test x$HAVE_GAVL = xtrue])
 
 AC_CHECK_PROG([DOXYGEN], [doxygen], [doxygen])
